models/custom-view/src/index.ts

```diff
@@ -2,10 +2,18 @@
 export * from './custom-view/types';
 export * from './custom-view-permission/types';
 export * from './custom-view-type-settings-for-custom-panel/types';
+export * from './custom-view-installation/types';
+export * from './custom-view-installation-permission/types';
+
+// Export constants
+export * from './custom-view/constants';
+export * from './custom-view-permission/constants';
 
 // Export models
 export * as CustomView from './custom-view';
 export * as CustomViewDraft from './custom-view/custom-view-draft';
 
 export * as CustomViewPermission from './custom-view-permission';
 export * as CustomViewTypeSettingsForCustomPanel from './custom-view-type-settings-for-custom-panel';
+export { RestrictedCustomViewInstallationForOrganization } from './custom-view-installation';
```

Comment:
This is kind of breaking the rule inside this barrel file but I grouped this specific `RestrictedCustomViewInstallationForOrganization` within `custom-view-installation` folder in case we ever need to add a model for `RestrictedCustomViewInstallationForProject`